### PR TITLE
Add New England mailing list

### PR DESCRIPTION
**Pull request by UnsolvedCypher**
Add a page for people to sign up for a mailing list of Northeast competitions. @timreyn and I will be using this to announce competitions as we've received multiple requests for this.

(Sorry for the accidental spam- I submitted a PR #149 too soon)

### CODE REVIEW

**Comment by jfly** [comment]
@UnsolvedCypher just FYI, you shouldn't have to create a new PR if it doesn't have what you want in it, you can just push to the branch you created the PR with.

File: cubingusa.py [hunk 1]
```diff
@@ -55,6 +55,7 @@
   webapp2.Route('/about/contact',
                 handler=ContactHandler(AppSettings.Get().contact_email, 'contact.html', 'CubingUSA'),
                 name='contact'),
+  webapp2.Route('/newengland', handler=BasicHandler('newengland.html'), name='newengland'),
```
Comment (jfly) [inline cubingusa.py:58]: I'd expect this diff to introduce a new file `newengland.html`. Did that file already exist, or did you possibly forget to add it to this PR?

Comment (UnsolvedCypher) [reply cubingusa.py:58]: You're right- I forgot to git add before committing. I just committed the actual html file.

**Comment by UnsolvedCypher** [comment]
@jfly Because I wanted to distinguish between the Northeast and New England (since I know NY is thinking about doing this as well), I wanted to rename the branch also. Is it considered a better practice to just make that distinction in the discussion and update the old PR?

**Comment by jfly** [comment]
> Is it considered a better practice to just make that distinction in the discussion and update the old PR?

I don't think there's any way to rename your branch without creating a new PR. That said, I don't think the branch name is terribly important (at least, as a code reviewer, I never ascribe any real meaning to it).

**Comment by jfly** [comment]
> Is it considered a better practice to just make that distinction in the discussion and update the old PR?

I don't think there's any way to rename your branch without creating a new PR. That said, I don't think the branch name is terribly important (at least, as a code reviewer, I never ascribe any real meaning to it).

**Comment by jfly** [comment]
> Is it considered a better practice to just make that distinction in the discussion and update the old PR?

I don't think there's any way to rename your branch without creating a new PR. That said, I don't think the branch name is terribly important (at least, as a code reviewer, I never ascribe any real meaning to it).

File: src/templates/newengland.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        <div class="card-title">
+          <h1>New England Mailing List</h1>
+        </div>
+        <div class="card-text">
+          <p>Note: this mailing list is operated by local Delegates, not by CubingUSA. CubingUSA is not responsible for this mailing list, and there is no guarantee that every competition will be posted here.</p>
+          <!-- Begin MailChimp Signup Form -->
+            <link href="//cdn-images.mailchimp.com/embedcode/classic-10_7.css" rel="stylesheet" type="text/css">
+            <style type="text/css">
+              #mc_embed_signup{background:#fff; clear:left; font:14px Helvetica,Arial,sans-serif; }
+              /* Add your own MailChimp form style overrides in your site stylesheet or in this style block.
```
Comment (jfly) [inline src/templates/newengland.html:17]: Do we want to follow this advice?

**Review by jfly — COMMENTED** [review]
Looks fine to me, but I don't know anything about mailchimp, and that's most of what this diff is about.

File: src/templates/newengland.html (hunk 2 above)
Comment (UnsolvedCypher) [reply src/templates/newengland.html:17]: To clarify, this is copy and pasted from MailChimp-generated code for the form. As for the style, I'm open to style suggestions but it actually fits in with the rest of the page surprisingly nicely (screenshot below)

newengland-mailing-list-screenshot

**Review by timreyn — APPROVED** [review]

**Pull request merged**